src/Orckestra.Composer/ComposerOvertureClient.cs

```diff
@@ -203,5 +217,16 @@ private static OvertureClientConfig GetClientConfig()
 
             return clientConfig;
         }
+
+        private void PreserveRequestType(Type requestType)
```

Comment:
This modification is likely not needed, as renaming operation be the same as first API call it makes to OCS is not good